examples/webgl_materials_cubemap_dynamic.html

```diff
@@ -74,10 +74,10 @@
 
 				//
 
-				material = new THREE.MeshBasicMaterial( {
+				material = new THREE.MeshStandardMaterial( {
```

Comment:
Switched the material for easier testing.

Reply:
Fine, but this example obfuscates bugs, which is why this bug was not detected.

In a later PR, perhaps this example can be improved -- maybe by adding an animate button, and performance stats.

Reply:
This should be fixed, too.
```js
cubeRenderTarget1 = new THREE.WebGLCubeRenderTarget( 256, {
	format: THREE.RGBFormat,
	generateMipmaps: false, // the default
	minFilter: THREE.LinearFilter // the default
} );
```

Reply:
I've ended up removing all options since we should move away from `RGBFormat`. Always using the RGBA path should be more performant across all devices since we avoid any kind of software emulation.

Reply:
Also added `stats'.js` to the example: f00fd4f639fdd035a2f3489e43ca7ef148ea5a5d